### ISSUE TITLE
Changes in V1.10.0

### ISSUE THREAD

**Issue by jgyates** [jgyates/genmon#177]
I will making a new release for version V1.10.0 and all new features will be listed, however I wanted to make a note about one specific change related to the start and stop mechanism. 

In previous version the startgenmon.sh program would start and stop both genmon.py and genserv.py and you could edit the file to make other apps start and stop. This worked, however if you updatd the software via the web interface then your changed startgenmon.sh file would be overwritten. To resolve this issue, starting with version 1.10.0 the program genloader.py was added and stargenmon.sh was modified to call genloader.py. 

In short, if you modified startgenmon.sh to load programs other than genmon.py and genserv.py, then you will need to edit the file /etc/genloader.conf (it will be copied to the /etc directory by genloader.py if it is not there) to enable the other programs you are using (like gensms.py, genpushover.py, etc).

Let me know if there are any questions.
Jason

**Comment by zekyl314** [comment]
Hi Jason,

Do you mean that the file is called genloader.conf instead of genmon.conf for the starting of the programs?

**Comment by jgyates** [comment]
I did mean /etc/genloader.conf. Thanks for catching this typo. I have updated the above comment.

**Comment by smeisner** [comment]
Is there ever a changelog generated with releases? I just upgraded from 1.10.7 to .12 and am not sure what the changes are.

**Comment by jgyates** [comment]
full releases are here https://github.com/jgyates/genmon/releases and they are documented with new features and bug fixes

incremental changes are checked in when a new feature is added or a bug is found and fixed.  For these updates you will need to view the comments of the checkin for details.

the version number is incremented for both incremental changes and full releases. In short full releases are accumulated incremental changes.

If you are curious about what has changed and don't want to upgrade unless you have read the change log, then wait for a full release to upgrade since the full releases have a change log with them.

**Comment by jgyates** [comment]
I added a change log here:

https://github.com/jgyates/genmon/blob/master/changelog.md

There will eventually be a link on the web interface for this.

**Comment by swvalenti194** [comment]
Upgraded to new version and see the add on tabs. Wont let me access settings tab?

**Comment by jgyates** [comment]
@swvalenti194 , please create a new issue for this. In the interim, if you have outgoing emails setup you can send me your log files by entering the following URL:

    yourpiaipaddress:8000/cmd/sendlogfiles

the above URL assumes you are not using secure HTTP (https), if you are just add /cmd/sendlogfiles to the end of the normal URL you use to access the web interface.

**Comment by swvalenti194** [comment]
Log files submitted

**Comment by jgyates** [comment]
Got your log files. Found a bug, just checked in a fix. Checkout the latest and try it out.

Also, according to your log, your weather is not working. That may be an invalid weather location ID. You can either disable the weather API setting or enter your city, state and country separated by commas, for example:

     Albany, NY, USA

**Comment by swvalenti194** [comment]
Cool thanks bro will give the weather a go and let you know. Much respect.

**Comment by swvalenti194** [comment]
upgraded still showing no settings

**Comment by jgyates** [comment]
Did you refresh your browser?

**Comment by jgyates** [comment]
Also, what browser are you using (Chrome, Firefox, etc)

**Comment by swvalenti194** [comment]
Chrome sir

**Comment by jgyates** [comment]
In chrome, go to settings menu, then More Tools > Developer Options, then try to access the settings page. Do you see any errors, if so send a screen shot. You can attach a screen shot on the desktop version of this thread (does not work on mobile web site). This will let me know if the error you are seeing is on the web page javascript or in the python code.

**Comment by swvalenti194** [comment]
Tried IE and same settings tab won't load

**Comment by jgyates** [comment]
Ok, we are close. 

Now use this URL to access the web page:

myipaddress:8000/index_verbose.html

Repeat the steps you took above and let me know the output.

**Comment by jgyates** [comment]
I am not 100% sure where the problem is yet, but I checked in a change that I think will help.

**Comment by swvalenti194** [comment]
No worries glad I can try and help. Don't feel like a total idiot at the moment.

**Comment by jgyates** [comment]
You are actually helping me a lot. :) 

Many thanks, I did some more testing and I feel more confident this may fix the problem. Let me know how the update goes.

**Comment by swvalenti194** [comment]
Setting page works now sir. Only question is now after fixing weather like you told me where should I see the weather display?

**Comment by swvalenti194** [comment]
any clue why when i do a reboot of pi it doesnt come back up and online?  I
have to go manually pull power than it comes up?

**Comment by jgyates** [comment]
That could be many things, including your wifi connection (assuming you are using wifi). How are you rebooting?

this command will reboot:

    sudo reboot

Can you ping your pi over the network when you reboot?

**Comment by swvalenti194** [comment]
I logged in via vnc viewer and reboot from GUI.

**Comment by jgyates** [comment]
Can you ping your pi over the network when you reboot?

**Comment by swvalenti194** [comment]
I didn't try that. Will do and report.

**Comment by swvalenti194** [comment]
Every time my AP disconnects etc the pi wont connect I have to restart the
pi by going to generator...I have multiple AP but it seems it just
associated with one in particular and won't find another how do I make sure
it scans?

pi@raspberrypi:~ $ sudo bash ./check_wifi.sh

Starting WiFi check for wlan0
Sat Oct 13 13:54:47 EDT 2018

Performing Network check for wlan0
Network is Okay

Current Setting:

pi@raspberrypi:~ $
